Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -762,6 +789,8 @@ WarpXOpenPMDPlot::SetupRealProperties (openPMD::ParticleSpecies& currSpecies,
         }
     }
 
+    // attributes need to be set only the first time BTD flush is called for a snapshot
+    if (isBTD and m_ParticleSetUp == false) return;
```

Comment:
Shouldn't this read 
```suggestion
    if (isBTD and m_ParticleSetUp == true) return;
```
so that it returns if the particle are already set up?

Reply:
note, just confusing name
```suggestion
    if (isBTD and m_doParticleSetup == false) return;
```